src/gui/tray/ActivityItemContent.qml

```diff
@@ -72,9 +72,22 @@ RowLayout {
             elide: Text.ElideRight
             font.pixelSize: Style.subLinePixelSize
             color: "#808080"
-        }
-    }
-
+        }  
+
+        Loader {
+            id: talkReplyTextFieldLoader
+            active: isChatActivity && isTalkReplyPossible
+
+            Layout.fillHeight: true
+            anchors.top: activityTextDateTime.bottom
```

Comment:
Avoid mixing anchors and layouts together -- in this case, the Column is not a layout, so the Layout property is not doing anything